docs/OV_Runtime_UG/stateful_models_intro.md

```diff
@@ -0,0 +1,90 @@
+Stateful models and State API {#openvino_docs_OV_UG_stateful_models_intro}
+==============================
+
+@sphinxdirective
+
+.. toctree::
+    :maxdepth: 1
+    :hidden:
+
+    openvino_docs_OV_UG_ways_to_get_stateful_model
+
+@endsphinxdirective
+
+## What is a Stateful Network?
```

Comment:
Introduction is good, but I would recommend to write overall approach why you will have variable and all those things in IR. Not necessarily in this chapter, but somewhere.
Do you expect that developer will convert model to this representation and will iteratively call model on different inputs? Then maybe should just say so.
Maybe showing difference in form of picture, i.e. when you have entire sequence fed vs. iteratively...